v2/observer/healthcheck.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/brigadecore/brigade/v2/internal/retries"
+)
+
+// runHealthcheckLoop checks Observer -> API Server comms
+func (o *observer) runHealthcheckLoop(ctx context.Context) {
+	ticker := time.NewTicker(o.config.healthcheckInterval)
+	defer ticker.Stop()
+
+	for {
+		if err := retries.ManageRetries(
```

Comment:
Retries on GETs are already built into the SDK... so you can lose the retry loop here.

Reply:
Oh yea, you'd mentioned this and I promptly forgot!  Will remove.

Reply:
No worries. Figure it's always refreshing when we get to delete code.